my-movie-list/src/components/Home/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { SearchForm } from "../SearchForm";
+import { MoviesResult } from "../MoviesResult";
+
+type MoviesResultProps = {
+    storeMovie: (movieToStore: Movie) => void;
+    setMoviesResultSearch: (movies: Movie[]) => void;
+    movies: Movie[];
+}
+
+export const Home = ({ storeMovie, setMoviesResultSearch, movies }: MoviesResultProps) => {
+    const [search, setSearch] = useState("");
+    const [searching, setSearching] = useState(false);
+    const [searchErro, setSearchErro] = useState(false);
+
+    return (
+        <>
+            <SearchForm search={search} setMoviesResultSearch={setMoviesResultSearch} setSearch={setSearch} setSearching={setSearching} setSearchErro={setSearchErro} />
+            {searching ? (<><h1>Buscando...</h1></>) : ""}
+            {searchErro ? (<><h1>Nenhum resultado encontrado</h1></>) : ""}
```

Comment:
Já que são valores booleanos e só retorna um item JSX, o que acha disso?
```suggestion
            {searching && (<h1>Buscando...</h1>)}
            {searchErro && (<h1>Nenhum resultado encontrado</h1>)}
```

O React interpreta bem esses valores booleanos dentro do JSX e, se for algo nulo, ou falso, não retorna nada pro DOM 😉